behaviors/index.js

```diff
@@ -0,0 +1,77 @@
+const utils = require('../utils');
+
+const greetBehavior = require('./greet');
+const helpBehavior = require('./help').helpBehavior;
+
+function parseMessage (message) {
+  return message.split(/[\s,.;:!?]/g).filter(String);
+}
+
+function messageResponse(botNick, parsed, behaviors) {
+  const behavior = behaviors.find(behavior =>
```

Comment:
Yes this looks ready -- perhaps one last thing could be a comment right here explaining exactly how behaviors are looked up, human readably. I know this will be in the readme too but it seems a little obscure on a quick read here.